aea/configurations/base.py

```diff
@@ -759,6 +760,15 @@ def __init__(
         self.dependencies = dependencies if dependencies is not None else {}
         self.description = description
 
+        # temporary solution till all protocols updated
```

Comment:
@DavidMinarsch we often do this kind of things to save backward compatibility. However, do we keep track of all of these temporary changes so to schedule them to be addressed at the right time?

Reply:
Not consistently, I agree we should though @marcofavorito .